src/xunit.runner.utility/Project/XunitFilters.cs

```diff
@@ -40,15 +47,30 @@ public XunitFilters()
         public Dictionary<string, List<string>> IncludedTraits { get; }
 
         /// <summary>
-        /// Gets the set of method filters for test classes to include.
+        /// Gets the set of class filters for test classes to exclude.
+        /// </summary>
+        public HashSet<string> ExcludedClasses { get; }
+
+        /// <summary>
+        /// Gets the set of class filters for test classes to include.
         /// </summary>
         public HashSet<string> IncludedClasses { get; }
 
+        /// <summary>
+        /// Gets the set of method filters for tests to exclude.
+        /// </summary>
+        public ICollection<string> ExcludedMethods => excludedMethods;
+
         /// <summary>
         /// Gets the set of method filters for tests to include.
         /// </summary>
         public ICollection<string> IncludedMethods => includedMethods;
 
+        /// <summary>
+        /// Gets the set of assembly filters for tests to exclude.
+        /// </summary>
+        public HashSet<string> ExcludedNameSpaces { get; }
```

Comment:
nit: "ExcludedNamespaces", according to https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/namespaces/using-namespaces

Reply:
Here I've stuck to the convention above, i.e. `IncludedNameSpaces` .

Reply:
I'll change it after merge.